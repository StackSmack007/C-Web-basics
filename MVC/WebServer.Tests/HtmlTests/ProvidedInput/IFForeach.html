@if('A'=='A')
{
    @foreach(int num in Enumerable.Range(1, 5))
    {
    <p>This is mambo Number @num</p>
    }
}