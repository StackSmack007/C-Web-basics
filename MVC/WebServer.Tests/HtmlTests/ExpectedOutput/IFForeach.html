<p>This is mambo Number 1</p>
<p>This is mambo Number 2</p>
<p>This is mambo Number 3</p>
<p>This is mambo Number 4</p>
<p>This is mambo Number 5</p>